examples/send_email_eu_region.rs

```diff
@@ -0,0 +1,21 @@
+use mailgun_v3::email::Message;
+use mailgun_v3::email::EmailAddress;
+use mailgun_v3::email::MessageBody;
+use mailgun_v3::Credentials;
+
+fn main(){
+    let msg = Message {
+        to: vec![EmailAddress::address("target@example.org")],
+        body: MessageBody::Text("hello world".to_string()),
+        subject: String::from("sample subject"),
+        ..Default::default()
+    };
+    let sender = EmailAddress::address("sender@example.org");
+    let creds = Credentials::with_base(
+        "http://api.eu.mailgun.net/v3",
```

Comment:
per https://documentation.mailgun.com/en/latest/api-intro.html#base-url it looks like this should be https, going to update it real quick for you